debian/build.go

```diff
@@ -354,6 +359,28 @@ func getKubeletCNIVersion(v version) (string, error) {
 	return fmt.Sprint("= 0.5.1"), nil
 }
 
+// getCRIToolsVersion assumes v coming in is >= 1.11.0-alpha.0
+func getCRIToolsVersion(v version) (string, error) {
+	sv, err := semver.Make(v.Version)
+	if err != nil {
+		return "", err
+	}
+
+	v1110, err := semver.Make("1.11.0-alpha.0")
+	if err != nil {
+		return "", err
+	}
+	v1120, err := semver.Make("1.12.0-alpha.0")
```

Comment:
it sounds like this might need to be v1.12.1, not v1.12.0.

Reply:
kubernetes/kubernetes#69033 has merged and is slated for v1.12.1. please update the version checks here.

Reply:
@chuckha Could you fix this asap? we're about to release v1.12.1 this Friday.